src/components/VideoDemo.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+// /////////////////////////////////////////////////////////////////
+import { OurDemo, UnderLine, OnlyVideoStyle } from './ui';
+// /////////////////////////////////////////////////////////////////
+export const VideoSection = () => (
+  <OurDemo>
+    <p> Checkout Our Demo </p>
+    <UnderLine />
+    <OnlyVideo />
```

Comment:
remove this and move content from this component here

Reply:
+